nonos/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.3.3"
+__version__ = "0.4"
```

Comment:
Don't break semantic versioning
```suggestion
__version__ = "0.4.0"
```